bin/varnishd/cache/cache_vrt_vmod.c

```diff
@@ -167,12 +167,14 @@ VPI_Vmod_Unload(VRT_CTX, struct vmod **hdl)
 {
 	struct vmod *v;
 
+	(void) ctx;
```

Comment:
If we no longer need `ctx` we might as well remove it from the function signature. Breaking VPI is safe by definition.